CoreTesting/src/au/gov/asd/tac/constellation/testing/debug/MemoryAction.java

```diff
@@ -31,16 +31,18 @@
 @Messages("CTL_MemoryAction=Memory")
 public final class MemoryAction implements ActionListener {
 
+    private final float divideBy = 1024;
+
     @Override
     public void actionPerformed(ActionEvent e) {
-        final long freemem = Runtime.getRuntime().freeMemory();
-        final long totalmem = Runtime.getRuntime().totalMemory();
-        final long maxmem = Runtime.getRuntime().maxMemory();
+        final float freemem = (Runtime.getRuntime().freeMemory() / divideBy) / divideBy;
+        final float totalmem = (Runtime.getRuntime().totalMemory() / divideBy) / divideBy;;
+        final float maxmem = (Runtime.getRuntime().maxMemory() / divideBy) / divideBy;;
```

Comment:
A very minor typo here but you have 2 semi-colons. Worth cleaning up now that its been picked up if that's ok?